stories/button/button.stories.js

```diff
@@ -41,40 +41,38 @@ export const primary = () => `
  */
 
 export const types = () => `<div class="fddocs-container fddocs-button-container"> 
-        <button class="fd-button">Default Button</button>
-        <button class="fd-button fd-button--emphasized">Emphasized Button</button>
-        <button class="fd-button fd-button--ghost">Ghost Button</button>
+        <button class="fd-button fd-button--emphasized">Primary Button</button>
+        <button class="fd-button">Secondary Button</button>
+        <button class="fd-button fd-button--ghost">Secondary Button</button>
         <button class="fd-button fd-button--positive">Positive Button</button>
         <button class="fd-button fd-button--negative">Negative Button</button>
         <button class="fd-button fd-button--attention">Attention Button</button>
-        <button class="fd-button fd-button--transparent">Transparent Button</button>
+        <button class="fd-button fd-button--transparent">Tertiary Button</button>
 </div>`;
 
 types.storyName = 'Button styles';
 types.parameters = {
     docs: {
-        storyDescription: `    
```

Comment:
I don't think it is a good idea to remove that. It would better to have both designs in parallel